webpack/envs/clientDevelopmentConfig.js

```diff
@@ -76,29 +69,18 @@ export const clientDevelopmentConfig = {
   },
   name: "force",
   plugins: [
-    new CaseSensitivePathsPlugin(),
     new webpack.HotModuleReplacementPlugin(),
-    new SimpleProgressWebpackPlugin({
-      format: "compact",
-    }),
     new ForkTsCheckerWebpackPlugin({
-      checkSyntacticErrors: true,
-      formatter: "codeframe",
-      formatterOptions: "highlightCode",
-      watch: ["./src"],
-    }),
-    new ForkTsCheckerNotifierWebpackPlugin({
```

Comment:
[Supported in 5](https://github.com/johnnyreilly/fork-ts-checker-notifier-webpack-plugin/blob/master/package.json#L42)

Reply:
Do we actually need system-level notifications? I have to manually disable these because I have too many notifications already.

Reply:
It's been in force for a while, it's helpful for folk who aren't in a terminal all day, who don't have their console open, and so on.

Reply:
Ooh, this sounds like an amazing opportunity for a survey!

Reply:
Do we have to go through this again? This pr is about upgrading webpack, not messing with our developer experience. Feel free to open a discussion about this after this has been upgraded, but stop trying to slip changes that mess with our DX.

Reply:
And feel free to guard this behind a flag as well

Reply:
Let me double-check that the Webpack warnings created by this plugin have been resolved and I will add it back. It was not compatible with prior versions of Webpack 5.

Reply:
Working added back! 🎉